apidoc/Titanium/App/iOS/iOS.yml

```diff
@@ -838,13 +838,19 @@ events:
         type: Dictionary
 
       - name: inBackground
-        summary: Boolean indicating if notification was received while app was in background. Available in Titanium SDK 6.2.0+.
+        summary: |
+          Boolean indicating if notification was received while app was in background. Available in Titanium SDK 6.2.0.
+          On iOS 10+ this is no longer available since this event will only fire if the app is in foreground.
         type: Boolean
 
       - name: threadIdentifier
         summary: |
-            The unique identifier for the thread or conversation related to this notification request.
-            It will be used to visually group notifications together. Available in Titanium SDK 7.4.0+ and iOS 12+.
+          The unique identifier for the thread or conversation related to this notification request.
+          It will be used to visually group notifications together. Available in Titanium SDK 7.4.0+ and iOS 10+.
```

Comment:
Property "threadIdentifier" is available in iOS 12+ . Please update.

Reply:
Why do we limit it to iOS 12+ only? It's available since iOS 10, isn't it?

Reply:
I guess this property was noticed only when iOS12's group notification came. So it was wrapped in impression of iOS 12  only property. Let's move it for iOS 10+ in implementation as well.